src/driver/amdxdna/aie2_smu.c

```diff
@@ -15,6 +15,9 @@
 #define AIE2_SMU_SET_SOFT_DPMLEVEL	0x7
 #define AIE2_SMU_SET_HARD_DPMLEVEL	0x8
 
+#define AIE2_SMU_SET_SOFT_HCLK_DPMLEVEL 0x7
+#define AIE2_SMU_SET_HARD_HCLK_DPMLEVEL 0x8
+
```

Comment:
@vengutta18 These two Macros was defined previously for the S0i3 issue.  Keep the latest one and replace the old one?

Reply:
I think I'll remove the latest ones, these came as part of the rebase